scripts/mkcloud

```diff
@@ -192,16 +192,19 @@ function error_exit
     exitcode=$1
     message=$2
     if [ -z "$SKIPSUPPORTCONFIG" ] ; then
+        local defaults="-l -I 5000"
+        local sc_cmd="supportconfig ${SUPPORTCONFIG_OPTIONS:-$defaults} | wc"
```

Comment:
Well actually it does work in the `ssh` case. But yes good catch that it doesn't work in the admin node case.  `eval` is totally evil and is overkill here :-)